src/app/components/overview-history-summary-header/overview-history-summary-header.component.html

```diff
@@ -0,0 +1,136 @@
+<div class="overview-summary-background">
+    <div
```

Comment:
This entire block and referred inputs seems to be an unneded copy-paste from repo-list component